Data ingestion using SINGER.py

```diff
@@ -20,11 +20,34 @@
 #exploring the data types of the variables declared above
 #note that since the below variable is a set data type, we must remember that we cant slice a set.
 #when we use the next() operator along with iter() operator on the users, the elements will be output
-#in random order since the set is not an ordered collection. so everytime we run the below code, the output will vary for the users variable
+#in random order since the set is not an ordered collection. so everytime we run the below code, the
+# output will vary for the users variables
 print("users is: ", type(users))
 print("accessing all elements of set by simply calling the name of the variable", next(iter(users)))
 
 print("cols is: ", type(cols))
 print("accessing all element of tuple using list's index notation", cols[0:3])
 
-print("json is: ", type(json))
\ No newline at end of file
+print("json is: ", type(json))
+
+#--
+#--
+
+# Complete the JSON schema
+schema = {'properties': {
+                    'items' : {
+                                'brand'         : {'type': 'string'},
+                                'model'         : {'type': 'string'},
+                                'price'         : {'type': 'number'},
+                                'currency'      : {'type': 'string'},
+                                'quantity'      : {'type': 'integer', 'minimum': 1},
+                                'date'          : {'type': 'string', 'format': 'date'},
+                                'countrycode'   : {'type': 'string', 'pattern': "^[A-Z]{2}$"},
+                                'store_name'    : {'type': 'string'}
+                                }
+                        }
+        }
+
+# Write the schema
+sg.write_schema(stream_name='products', schema=schema, key_properties=[])
```

Comment:
Good note that you can declare the json variable name without single or double quotes